Process.c

```diff
@@ -192,25 +192,24 @@ static inline void Process_writeCommand(const Process* this, int attr, int basea
          if (comm[i] == '/') {
             basename = i + 1;
          } else if (comm[i] == ':') {
-            finish = i + 1;
+            len = i + 1;
             break;
          }
       }
-      if (!finish) {
+      if (len == 0) {
          if (this->settings->showProgramPath) {
             start += basename;
          } else {
             comm += basename;
          }
-         finish = this->basenameOffset - basename;
+         len = this->basenameOffset - basename;
       }
-      finish += start - 1;
```

Comment:
Not sure about this change. Can you explain?

Reply:
Previously we needed the finish index for `RichString_setAttrn()`. So the length had to be adjusted by the starting point (`+ start`) and since the finish index meant the last character index to change (and not 1 past the last one) `1` had to be subtracted.
Now that's obsolete.